apps/sms/views/inbox/js/inbox.js

```diff
@@ -974,7 +974,7 @@ var InboxView = {
     }
 
     if (thread.isDraft) {
-      this.removeThread(thread.id);
+      this.deleteThread(thread.id);
```

Comment:
Yeah I think we can fire another bug to improve the thread deletion. BTW maybe we can rename the removeThread as well because it's misleading, removeThreadDom or something could be more clear.